com.unity.multiplayer.mlapi/Tests/Runtime/Metrics/WaitForMetricValues.cs

```diff
@@ -12,6 +12,8 @@ public class WaitForMetricValues<TMetric> : IMetricObserver
     {
         private readonly string m_MetricName;
         private bool m_Found = false;
+        private bool m_HasError = false;
```

Comment:
I didn't make most of the changes in this file so I think this may be a merge issue, but if they should stay please let me know otherwise i'll revert them

Reply:
Yeah, that's a problematic merge.

The HasError stuff shouldn't be there anymore.

Reply:
Cool I will pick out what shouldn't be there!